disk.go

```diff
@@ -13,31 +13,48 @@ const (
 	DqdChunks
 )
 
+var idCounter int
 //==================================================================
```

Comment:
Instead of global static, you could make disk ID a function of the r.GetID(), where r is the runner passed into disk constructor, line 42